crates/burrego/src/opa/evaluation_result.rs

```diff
@@ -0,0 +1,6 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize)]
+pub struct EvaluationResult {
+    pub result: bool,
+}
```

Comment:
I think this is limiting. Violations can return also explanation messages about why something has been rejected

Reply:
Yes, the main reason I didn't enter in detail here is: do we want to have this structure on the consumable crate, or our API is meant  to return `serde_json`? I prefer this approach, and I think I can expand this struct to the documented fields -- have to check where it's documented.